src/apps/survey/components/main/final/Final.hooks.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  useSurveyFinalListQuery,
+  useSurveyListByIdQuery,
+  useSurveyMainListQuery,
+} from "@/apps/survey/queries";
+import { useMoveOtherContents, useShareLink } from "@/hooks";
+import { useState } from "react";
+import { ChartDataStore } from "./Final.types";
+
+export const useGetSurvey = () => {
+  const { data: surveyMainList } = useSurveyMainListQuery();
+  const { data: surveyFinalList } = useSurveyFinalListQuery(true);
+  const { data: surveyListById } = useSurveyListByIdQuery();
```

Comment:
```suggestion
  const { data: surveyInfo } = useSurveyListByIdQuery();
```

정도가 data를 나타내기 적합한 용어같네용 더이상 ById라는 맥락은 필요없어졌으니까요 data에는